samples/Plugin.Maui.Audio.Sample/ViewModels/AudioRecorderPageViewModel.cs

```diff
@@ -52,6 +55,84 @@ public bool IsRecording
 		this.dispatcher = dispatcher;
 	}
 
+	ChannelTypesViewModel selectedChannelType;
+	public ChannelTypesViewModel SelectedChannelType
```

Comment:
I don't think we need the ViewModel wrapper for this or the other engine, if we are just calling ToString on the values then MAUI will do that for us

Reply:
I usually do that because I then have more complex options like using a converter with app resources or to add an IsSelected property, but maybe it's overkill here 😊